server/config/config.go

```diff
@@ -965,6 +965,20 @@ func (c *PDServerConfig) Clone() *PDServerConfig {
 	}
 }
 
+// Validate is used to validate if some pd-server configurations are right.
+func (c *PDServerConfig) Validate() error {
+	switch c.DashboardAddress {
+	case "auto":
```

Comment:
what mean "auto" and "none"

Reply:
A specific address represents a specific instance of the running dashboard. `none` represents no instance that has dashboard running, which means that the dashboard is stopped. "auto" means that PD can later decide on an instance of the running dashboard, and after PD is run, it will be replaced with a specific address.

Reply:
[Chinese Docs](https://docs.pingcap.com/zh/tidb/v4.0/dashboard-ops-deploy#%E5%88%87%E6%8D%A2%E5%85%B6%E4%BB%96-pd-%E5%AE%9E%E4%BE%8B%E6%8F%90%E4%BE%9B-tidb-dashboard-%E6%9C%8D%E5%8A%A1)

Reply:
do we need to comment on them to code?

Reply:
There is a comment at the definition: `// There are some values supported: "auto", "none", or a specific address, default: "auto"`